src/main/java/com/openshift/jenkins/plugins/pipeline/dsl/OpenShiftExec.java

```diff
@@ -0,0 +1,196 @@
+
+public class OpenShiftExec extends OpenShiftBaseStep implements IOpenShiftExec {
+
+    protected String pod;
+    protected String container;
+    protected String command;
+    protected List<Argument> arguments = new ArrayList<>();
+    protected String waitTime;
+
+    @DataBoundConstructor
+    public OpenShiftExec( String pod ) {
+        this.pod = pod;
+    }
+
+    @DataBoundSetter
+    public void setContainer(String container) {
+        this.container = container;
+    }
+
+    @DataBoundSetter
+    public void setCommand(String command) {
+        this.command = command;
+    }
+
+    @DataBoundSetter
+    public void setArguments(List<Argument> arguments) {
+        this.arguments = arguments;
+    }
+
+    @DataBoundSetter
+    public void setWaitTime(String waitTime) {
+        this.waitTime = waitTime;
+    }
+
+    public String getPod() {
+        return pod;
+    }
+
+    public String getContainer() {
+        return container;
+    }
+
+    public String getCommand() {
+        return command;
+    }
+
+    public List<Argument> getArguments() {
+        return arguments;
+    }
+
+    public String getWaitTime() {
+        return waitTime;
+    }
+
+    public String getWaitTime(Map<String,String> overrides) {
+        String val = getOverride(getWaitTime(), overrides);
+        if (val.length() > 0)
+            return val;
+        return Long.toString(GlobalConfig.getBuildWait());
+    }
+
+	@Extension
+    public static class DescriptorImpl extends AbstractStepDescriptorImpl {
+
+        public DescriptorImpl() {
+            super(OpenShiftExecExecution.class);
+        }
+
+        @Override
+        public String getFunctionName() {
+            return "openshiftExec";
+        }
+
+        @Override
+        public String getDisplayName() {
+            return DISPLAY_NAME;
+        }
+
+        private Object requiredArgument( Map<String, Object> arguments, String arg ) {
+            Object o = arguments.get( arg );
+            if ( o == null ) {
+                throw new IllegalArgumentException( "Missing required argument: " + arg );
+            }
+            return o;
+        }
+
+        private String argumentAsString(Map<String, Object> arguments, String arg, String def ) {
+            Object o = arguments.get( arg );
+            if ( o == null ) {
+                return def;
+            }
+            return o.toString();
+        }
+
+        private String argumentAsString(Map<String, Object> arguments, String arg ) throws IllegalArgumentException {
+            return requiredArgument(arguments, arg ).toString();
+        }
+
+        @Override
+        public Step newInstance(Map<String, Object> arguments) throws Exception {
+            OpenShiftExec step = new OpenShiftExec( argumentAsString( arguments, "pod" ) );
+            step.setContainer( argumentAsString( arguments, "container", "" ) );
+
+            Object commandObject = requiredArgument( arguments, "command" );
```

Comment:
Hmmmmmmmmmmmmmm :-) 

Just to make sure I understand, is it because only 1 constructor signature can have the @DataBoundConstructor annotation?

If so, then yep, I'm good with this exception.